activities.html

```diff
@@ -10,10 +10,17 @@
 			<h2 class="animated fadeIn">Website Activity</h2>
 			<p class="lead mb64">Things to work on with</p>
 		</div>
+		<div class="col-sm-12">
+			<div class="form-group">
+			<input type="text" id="searchString" class="form-control" />
+			<div id="issueSearchOutput"></div>
+			</div>
+		</div>
 		<div class="col-sm-12 col-md-12 text-center mentor-list" id="issues">
 			<div class="issue-container container"></div>
 		</div>
 	</div>
 </section>
 
 {% include footer.html %}
+				
```

Comment:
You got spaces here

Reply:
something big coming up. ignore the PR for a few minutes 🔥